cmd/manager/main.go

```diff
@@ -137,6 +138,11 @@ func main() {
 		os.Exit(1)
 	}
 
+	if err := configv1.AddToScheme(mgr.GetScheme()); err != nil {
```

Comment:
What does this do?

Reply:
Hmmm, I was getting this error and I thought the scheme needed to be added, but it sees it was a part of my wrong approach...  I commented it out, but I don't get this error any more.  I'm removing this diff.
`../../vendor/github.com/openshift/client-go/config/clientset/versioned/typed/config/v1/config_client.go:132:45: scheme.Codecs.WithoutConversion undefined (type serializer.CodecFactory has no field or method WithoutConversion)`

Reply:
My test mistake... :(  This change was needed.  Without it, getting cluster proxy fails with this error.
`Failed to get proxy: no kind is registered for the type v1.Proxy in scheme "k8s.io/client-go/kubernetes/scheme/register.go:61"`
Opening a pr to resurrect it...